### PR TITLE
Update project to Swift 5 and xCode11

### PR DESCRIPTION
**Pull request by SergioEstevao**
To test:
 - Check if the project compiles and works correctly in Xcode 11.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,24 +9,24 @@ workflows:
     jobs:
```
Comment (jtreanor) [inline .circleci/config.yml]: You will need to update the orb version above this to `ios: wordpress-mobile/ios@0.0.32` so it has the needed fixes.

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,24 +9,24 @@ workflows:
...
           name: Test
-          xcode-version: "10.2.0"
+          xcode-version: "11.0"
           carthage-update: true
           carthage-working-directory: Example
           bundle-install: false
           pod-install: false
           workspace: Aztec.xcworkspace
           scheme: AztecExample
           device: iPhone XS
```
Comment (jtreanor) [inline .circleci/config.yml]: This should be `iPhone 11`. Xcode 11 does not come with an iPhone XS simulator by default.

**Comment by jkmassel** [comment]
One other issue – in order to compile this, we'll need to update Gridicons as well. I can take care of that – I'll tag you in the dependent PR?

**Comment by SergioEstevao** [comment]
@jtreanor I followed your tips, but I'm still having trouble with CircleCI... any more ideas?

**Comment by jtreanor** [comment]
@SergioEstevao It looks like you will need to update CocoaPods to at least 1.7.x for Xcode 11 compatibility.

You can update [this line](https://github.com/wordpress-mobile/AztecEditor-iOS/blob/develop/Gemfile#L2) to `1.7.5` and run `bundle install`

**Comment by SergioEstevao** [comment]
@jtreanor it worked! Do you mind review it now?

**Pull request merged**